src/MoBi.Core/MoBi.Core.csproj.orig

```diff
@@ -0,0 +1,68 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I won't keep commenting delete for these files of type ```.orig```. They should all be deleted.